skiko/build.gradle.kts

```diff
@@ -277,20 +277,21 @@ tasks.withType(CppCompile::class.java).configureEach {
 project.tasks.register<Exec>("objcCompile") {
     val inputDir = "$projectDir/src/jvmMain/objectiveC/${targetOs.id}"
     val outDir = "$buildDir/objc/$target"
-    val objcSrc = "drawlayer"
+    val names = File(inputDir).listFiles()!!.map { it.name.removeSuffix(".m") }
+    val srcs = names.map { "$inputDir/$it.m" }.toTypedArray()
+    val outs = names.map { "$outDir/$it.o" }.toTypedArray()
+    workingDir = File(outDir)
     commandLine = listOf(
         "clang",
         "-mmacosx-version-min=10.13",
         "-I$jdkHome/include",
         "-I$jdkHome/include/darwin",
         "-c",
-        "$inputDir/$objcSrc.m",
```

Comment:
What is output directory then? Why not compile with multiple clang invocations?

Reply:
Output directory is set above:
```
workingDir = File(outDir)
```
clang doesn't support setting output directory for multiple source files via arguments